prowler/compliance/aws/cis_1.5_aws.json

```diff
@@ -533,7 +533,7 @@
       "Id": "2.1.5",
       "Description": "Ensure that S3 Buckets are configured with 'Block public access (bucket settings)'",
       "Checks": [
-        "s3_account_level_public_access_blocks"
+        "s3_bucket_level_public_access_block"
```

Comment:
While I requested this check, I do not think it should be added to the CIS compliance file. Generally account level BPA settings are a better way to achieve CIS compliance. The `s3_bucket_level_public_access_block` check is extremely useful for edge cases (where an account has a legitimate need for a public bucket and a risk exception has been made), but the account level checks are a preferable approach for most environments / AWS accounts.